java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -1636,4 +1637,29 @@ public void zcard_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(value, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void zscore_returns_success() {
+        // setup
+        String key = "testKey";
+        String member = "testMember";
+        String[] arguments = new String[] {key, member};
+        Double value = 3.5;
+
+        CompletableFuture<Double> testResponse = mock(CompletableFuture.class);
+        when(testResponse.get()).thenReturn(value);
```

Comment:
Please change to a real CompletableFuture to avoid the warning.  
```
        CompletableFuture<Double> testResponse = new <>CompletableFuture();
        testResponse.complete(value);
```


examples: https://github.com/Bit-Quill/glide-for-redis/pull/163/files